wlc/__init__.py

```diff
@@ -86,48 +141,56 @@ def process_error(error):
                 raise WeblateDeniedError()
 
             reason = error.response.reason
-            raise WeblateException(f"HTTP error {status_code}: {reason}")
-
-    def raw_request(self, method, path, params=None, files=None):
+            error_string = ""
+            try:
+                error_string = str(error.response.json())
+            except Exception:
+                pass
```

Comment:
This makes the code intention much easier to understand:

```suggestion
            try:
                error_string = str(error.response.json())
            except Exception:
                error_string = ""
```